app/templates/flash-message-change-password-bug.html

```diff
@@ -0,0 +1,5 @@
+<div class="pull-right">
+    <button class="btn btn-default btn-primary submit" ng-click="$parent.startRecoveryProcess()">Go to Settings</button>
+</div>
+<span class="flash-info">Your recovery code has an error.</span>
```

Comment:
We should probably get a link to a help desk faq entry, or some more explanatory info here.